packages/file_testing/pubspec.yaml

```
@@ -1,13 +1,13 @@
 name: file_testing
-version: 2.1.0
+version: 2.2.0
 authors:
 - Todd Volkert <tvolkert@google.com>
 description: Testing utilities for package:file
 homepage: https://github.com/google/file.dart/packages/file_testing
 
 dependencies:
   meta: ^1.1.2
-  test_api: ^0.2.2
+  test: ">=1.0.0 <2.0.0"
```

The equivalent and simpler format is:

```suggestion
  test: ^1.0.0
```